### PR TITLE
[T5A2][W11-B3]Fan Wegiuang

### PR DESCRIPTION
**Pull request by fanwgwg**
Ready for review
The first two commits should be ignored, because I forgot to checkout master before creating a new branch,

### CODE REVIEW

File: src/seedu/addressbook/commands/AddCommand.java [hunk 1]
```diff
@@ -58,13 +58,18 @@ public ReadOnlyPerson getPerson() {
...
-    public CommandResult execute() {
+    public CommandResult execute() throws Exception{
         try {
             addressBook.addPerson(toAdd);
             return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
         } catch (UniquePersonList.DuplicatePersonException dpe) {
             return new CommandResult(MESSAGE_DUPLICATE_PERSON);
         }
     }
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/AddCommand.java]: You forgot to put the header comment. For this kind of method you may not use `@return`, instead you can state it in the description like `Returns true if ...`, and don't forget to give complete explanation like `However, adding a duplicate person will not mutate data but the corresponding AddCommand object will still return true for this method.`

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments. 
Good job.. Keep up the good work.

File: src/seedu/addressbook/commands/AddCommand.java (hunk 1 above)
Comment (fanwgwg) [inline src/seedu/addressbook/commands/AddCommand.java]: Yes. May I clarify with youthat for what kind of methods I should use Javadoc comments? Cuz I didn't find it in the coding standard provided by the handbook.

Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/AddCommand.java]: I guess it's mentioned that for a very simple method you may not put the Javadoc comment. However, for this case you need to add the comment to mention that if you add a duplicate person it won't mutate the data but still return true, just to let other people know the behavior and also that you already consider this case. :)

Comment (fanwgwg) [inline src/seedu/addressbook/commands/AddCommand.java]: Okay I understand. Thank you very much!

**Comment by fanwgwg** [comment]
Comments reviewed. Thanks!

**Pull request closed without merging**